Dockerfile

```diff
@@ -3,6 +3,13 @@ FROM circleci/python:3.8-buster
 # Base image uses "circleci", to avoid using `sudo` run as root user
 USER root
 
+# Golang env flags that limit parallel execution
+# The golang default is to use the max CPUs or default to 36.
+# In CircleCI 2.0 the max CPUs is 2 but golang can't get this from the environment so it defaults to 36
+# This can cause build flakiness for larger projects. Setting a value here that can be overridden during execution
+# may prevent others from experiencing this same problem.
+ENV GOFLAGS=-p=4
```

Comment:
You mentioned in the thread setting ENV GOFLAGS=-p=2, not 4. Why in the end did you go with 4? 
I'm guessing that it may be still considered a "sane" default with wiggle room for those with a bigger CircleCI container?

Reply:
I ended up testing with 4 on MilMove and it seems to be totally fine. I was feeling a bit scared of setting it down too low at 2. 🙃  Good question though.